recipe/meta.yaml

```
@@ -47,6 +47,7 @@ outputs:
         - python >=3.9
       run:
         - python >=3.9
+        - openff-nagl-models
```

```suggestion
        - openff-nagl-models >= 0.1.0
```
This would probably work just as well, or better, I have no opinion